### ISSUE TITLE
Bash-like tab completion causes Argument Out Of Range Exception

### ISSUE THREAD

**Issue by alexschneider** [PowerShell/PSReadLine#1051]
I used the stackoverflow here (https://stackoverflow.com/questions/8264655/how-to-make-powershell-tab-completion-work-like-bash) to change my auto-completion style. The output I saw for file auto-complete is this:

```powershell
Oops, something went wrong.  Please report this bug with the details below.                                            
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new                                                      
-----------------------------------------------------------------------                                                
Last 20 Keys:                                                                                                           
G e t 0 Backspace - F i l e H a s h Tab Space . \ Tab Tab                                                                                                                                                                                    
Exception:                                                                                                             
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.                                                                                                
Parameter name: top                                                                                                    
Actual value was -17.                                                                                                     
at System.Console.SetCursorPosition(Int32 left, Int32 top)                                                             
at Microsoft.PowerShell.PSConsoleReadLine.Menu.EnsureMenuAndInputIsVisible(IConsole console, Int32 tooltipLineCount)   at Microsoft.PowerShell.PSConsoleReadLine.Menu.DrawMenu(Menu previousMenu)                                             
at Microsoft.PowerShell.PSConsoleReadLine.PossibleCompletionsImpl(CommandCompletion completions, Boolean menuSelect)   at Microsoft.PowerShell.PSConsoleReadLine.CompleteImpl(Boolean menuSelect)                                             
at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)                                                                                                 
at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()                                                                  
at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)            
-----------------------------------------------------------------------
PS C:\Users\Alex> Get-FileHash .\
```

Environment data
----------------

```powershell
PS version: 5.1.18362.145                                                                                              
PSReadline version: 2.0.0-beta2
os: 10.0.18362.1 (WinBuild.160101.0800)
PS file version: 10.0.18362.1 (WinBuild.160101.0800)
BufferWidth: 119
BufferHeight: 27
```
-->

Steps to reproduce or exception report
--------------------------------------
1. Use Windows Terminal
2. Run `Set-PSReadlineKeyHandler -Key Tab -Function Complete`
3. Type `Get-FileHash .\` in a directory with files in it
4. Hit tab

This does not repro when running powershell the normal manner.

**Comment by msftrncs** [comment]
I cannot reproduce this with the latest version, 2.0.0-beta5.  can you try that, @alexschneider?

**Comment by daxian-dbw** [comment]
@alexschneider, the issue should have been fixed in 2.0.0-beta5. Please upgrade to that version.
